### PR TITLE
Added Peter Ferguson and Erik Dennihy

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

**Review by psferguson — APPROVED** [review]

**Review by arunkannawadi — APPROVED** [review]

File: etc/authordb.yaml [hunk 1]
```diff
@@ -1358,6 +1372,13 @@ authors:
     initials: Bryce
     name: Kalmbach
     orcid: 0000-0002-6825-5283
+  kanawadia:
```
Comment (arunkannawadi) [inline etc/authordb.yaml:1375]: I _think_ the username is correct though. I remember having an unfortunate typo

Comment (leannep) [reply etc/authordb.yaml:1382]: I can update this as well

File: etc/authordb.yaml [hunk 2]
```diff
@@ -2077,6 +2084,13 @@ authors:
     initials: Andrew P.
     name: Rasmussen
     orcid: null
+  reeds:
+    affil:
+    - Princeton-Astro
+    altaffil: []
+    initials: Sophie
```
Comment (sr525) [inline etc/authordb.yaml:2098]: I use Sophie L Reed because Scott Reed publishes too many papers on soil and makes me feel bad.

Comment (leannep) [reply etc/authordb.yaml:2098]: OK, will correct

**Pull request merged**